nav2_bringup/launch/nav2_simulation_launch.py

```diff
@@ -108,19 +109,20 @@ def generate_launch_description():
         cmd=[simulator, '-s', 'libgazebo_ros_init.so', world],
         cwd=[launch_dir], output='screen')
 
-    start_robot_state_publisher_cmd = launch.actions.ExecuteProcess(
-        cmd=[
-            os.path.join(
-                get_package_prefix('robot_state_publisher'),
-                'lib/robot_state_publisher/robot_state_publisher'),
-            os.path.join(
-                get_package_share_directory('turtlebot3_description'),
-                'urdf', 'turtlebot3_waffle.urdf'),
-            ['__params:=', configured_params]],
-        cwd=[launch_dir], output='screen')
+    urdf = os.path.join(
+        get_package_share_directory('turtlebot3_description'), 'urdf', 'turtlebot3_waffle.urdf')
+
+    start_robot_state_publisher_cmd = launch_ros.actions.Node(
+            package='robot_state_publisher',
+            node_executable='robot_state_publisher',
+            node_name='robot_state_publisher',
+            output='screen',
+            parameters=[configured_params],
+            arguments=[urdf])
 
     start_rviz_cmd = launch.actions.ExecuteProcess(
         cmd=[os.path.join(get_package_prefix('rviz2'), 'lib/rviz2/rviz2'),
+            [ "__log_level:=fatal"],
```

Comment:
This would mask alot of problems, you sure we want to do that? I just checked the rviz codebase, there are no `FATAL` calls

Reply:
Ops, that escaped, I usually add this to my RVIZ launch for a cleaner terminal (and since most of the current errors are well known). I'll remove it.